src/app/components/Navbar.tsx

```diff
@@ -0,0 +1,177 @@
+"use client";
+
+import Image from "next/image";
+import { useState } from "react";
+import Link from "next/link";
+
+export default function Navbar() {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  // Toggle menu visibility
+  const toggleMenu = () => setIsMenuOpen(!isMenuOpen);
+
+  return (
+    <nav className="bg-[#1E3432] text-white py-4 rounded-full shadow-lg w-[90%] mx-auto mt-6 relative z-50">
+      <div className="container mx-auto flex justify-between items-center px-4 md:px-8">
+        {/* Left Section (Hamburger Icon for Mobile) */}
+        <div
+          className="md:hidden flex items-center ml-10"
+          onClick={toggleMenu}
+          role="button" // Define as a button for accessibility
+          tabIndex={0} // Make it focusable
+          onKeyUp={(e) => {
+            if (e.key === "Enter" || e.key === " ") {
+              // Handle Enter and Space keys for accessibility
+              toggleMenu();
+            }
+          }}
+        >
+          {isMenuOpen ? (
+            <div className="space-y-2">
+              <div className="w-6 h-0.5 bg-[#F2B263] mb-1"></div>
+              <div className="w-6 h-0.5 bg-[#F2B263] mb-1"></div>
+            </div>
+          ) : (
+            <div className="space-y-2">
+              <div className="w-6 h-0.5 bg-[#F2B263]"></div>
+              <div className="w-6 h-0.5 bg-[#F2B263]"></div>
+              <div className="w-6 h-0.5 bg-[#F2B263]"></div>
+            </div>
+          )}
+        </div>
+
+        {/* Center Section (Logo) */}
+        <div
+          className={`flex justify-center md:justify-start flex-1 ${isMenuOpen ? "hidden" : ""}`}
+        >
+          <Image
+            src="/images/LOGO.png"
+            alt="Currenci Logo"
+            className="w-28 sm:w-34 md:w-36 lg:w-42 xl:w-48 h-auto"
+            width={150} // Set appropriate width
+            height={150} // Set appropriate height
+          />
+        </div>
+
+        {/* Right Section (Login Button for Desktop) */}
+        <div className="hidden md:flex items-center gap-6 md:gap-8">
+          {/* Links */}
+          <Link
+            href="/about"
```

Comment:
create a array of the navlinks and map over them. your code will unnecessarily increase the codebase size.